### DISCUSSION TITLE
Cannot parse "serverless.yml": bad indentation of a mapping entry in  <filepath>

### DISCUSSION THREAD

**Discussion by krupahirani** [serverless/serverless#11370] [Q&A]
When i deploy the node js api to lambda i'm getting this error. How to solve it?

**Comment by nponeccop** [comment]
1. Ensure that the indentation is valid (e.g. look for mixing tabs and spaces). 
2. Remove parts of the config until the problem disappears
3. Prepare a small example `serverless.yaml` that doesn't work and doesn't contain sensitive info, and post it here. It's hard to answer without a file.
